packages/examples/cvat/exchange-oracle/src/core/tasks/skeletons_from_boxes.py

```diff
@@ -18,6 +18,7 @@
 @frozen(kw_only=True)
 class RoiInfo:
     original_image_key: int
+    original_image_id: str
```

Comment:
I can't find where this field is used. Is it only for debugging?

Reply:
It was used before refactoring but doesn’t seem to be needed now